server/migrations/20220203231624-fix-ion.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.bulkUpdate('Chains', {
+        symbol: 'ION',
+        network: 'ion',
+        base: 'cosmos',
+        bech32_prefix: 'ion',
```

Comment:
It doesn't get its own bech32_prefix because it's still an offchain community. It should be `osmo`.

Reply:
👍